gasCosts/gasCosts.js

```diff
@@ -19,7 +19,9 @@ import {
 } from "../helpers/constants";
 import { getTokenArgs, currentBlockTime, createSignatures, forwardTime } from "../helpers/test-helper";
 import { setupColonyVersionResolver } from "../helpers/upgradable-contracts";
-import { giveUserCLNYTokens } from "../helpers/test-data-generator";
+import { giveUserCLNYTokensAndStake } from "../helpers/test-data-generator";
+import ReputationMiningClient from "../client/main";
```

Comment:
This is the only point where I see a little problem with separating out the client.  But it should be possible and would make it much cleaner especially for a reasonable separation of concerns.

Maybe at some point someone could walk me through what's actually being tested here.